### ISSUE TITLE
Remove remaining eos.io references from docs portal

### ISSUE THREAD

**Issue by nsjames** [eosnetworkfoundation/devrel#193]
We want to remove all remaining references to `eos.io` 

https://docs.google.com/spreadsheets/d/1IYbcQFBV1DPCvhcxJZcgzzqwK7PaC2hCywgOMi2fxiM/edit#gid=711758493

AC:
- Identify where remaining links to `eos.io` are
- Replace all references to `eos.io` with our docs URLs
  - Ensure you are using the right branch that the docsgen is using (check scripts)
- Ask Val to recrawl staging after merges

Note: Check repositories that are used by the docsgen: https://github.com/eosnetworkfoundation/docsgen/tree/main/scripts

**Comment by lparisc** [comment]
Old directories previously deployed on the AWS server still remain after being deleted from the source repos. For instance, there was still a `welcome` folder produced from building the old `welcome` repo (now renamed `docs`). Therefore, I had to remove some of these dead folders directly from the AWS web server to get rid of some of these `eos.io` references. Other folders that needed to be removed manually: `ko`, `zh`.

**Comment by lparisc** [comment]
Removed last pesky `eos.io` references from docs portal. Notes: with the exception of the swift reference, regenerated successfully after replacing the `eos.io` links, I ended up replacing most `eos.io` links directly within the AWS production site. Rationale: both staging and production site reuse most repo branches, so rebuilding production would produce a mixed production/staging site w/ inaccurate contents/links. We need a way to clearly specify default repo/branches to use for staging vs. production, and allow overrides passed to the `clean_rebuild.sh` script; added item to agenda to discuss in next team's meeting.
